relay-rust/scripts/gnirehtet-run.cmd

```diff
@@ -1,2 +1,3 @@
+@adb shell am start -n com.genymobile.gnirehtet/.AuthorizationActivity

```

Comment:
The approach I had in mind to solve the [nodialog](https://github.com/Genymobile/gnirehtet/issues?q=is%3Aopen+is%3Aissue+label%3Anodialog) issues (before Android Q) was to replace the broadcast receiver (`GnirehtetControlReceiver`) itself by an activity (`GnirehtetActivity`).

I guess this would also solve the Android Q issue?

Reply:
If you change GnirehtetControlReceiver to GnirehtetActivity, I think it will work well.
I also add GnirehtetControlReceiver's feature in AuthorizationActivity.